modules/cluster-agent/internal/server/server_test.go

```diff
@@ -146,6 +146,8 @@ func TestNewServerTLSConfiguration(t *testing.T) {
 				assert.Error(t, err)
 				if tt.errContains != "" {
```

Comment:
It looks like `errContains` isn't being used anymore. If that's the case then let's remove it.